### PR TITLE
3.3.4: add metric summary doc

### PR DESCRIPTION
**Pull request by crazycs520**
Signed-off-by: crazycs <crazycs520@gmail.com>

### CODE REVIEW

File: session3/chapter3/metrics-summary.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
+# 监控汇总表
+
+由于 TiDB 的监控表太多，每次查询一项监控都去执行一次 SQL 查询非常的不方便，这就是监控汇总表的由来。
```
Comment (lonng) [inline session3/chapter3/metrics-summary.md]:
```suggestion
由于 TiDB 的监控表太多，TiDB 4.0 提供了监控汇总表，来减少使用 SQL 对各个监控指标进行逐项查询的工作量。
```

File: session3/chapter3/metrics-summary.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+监控汇总表 `information_schema.metrics_summary` 和 `information_schema.metrics_summary_by_label` 用于汇总所有监控数据。两者不同在于 information_schema.metrics_summary_by_label 会按照 label 进行汇总。
+
+## 查询示例
+
+查询 `metrics_summary` 监控汇总表，在  `['2020-03-08 13:23:00', '2020-03-08 13:33:00')`  时间范围内，TiDB 的 Top 3 的 P99 的平均最长耗时。这里查询监控汇总表是通过 `/*+ time_range() */`  hint 来指定查询时间范围。
```
Comment (lonng) [inline session3/chapter3/metrics-summary.md]:
```suggestion
查询 `metrics_summary` 监控汇总表，在  `['2020-03-08 13:23:00', '2020-03-08 13:33:00')`  时间范围内，TiDB 的 Top 3 的 P99 的平均最长耗时。这里查询监控汇总表是通过 `/*+ time_range() */` hint 来指定查询时间范围。
```

File: session3/chapter3/metrics-summary.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+***************************[ 3. row ]***************************
+METRICS_NAME | tidb_kv_request_duration
+QUANTILE     | 0.99
+SUM_VALUE    | 0.170232
+AVG_VALUE    | 0.004601
+MIN_VALUE    | 0.000975
+MAX_VALUE    | 0.013
+COMMENT      | The quantile of kv requests durations by store
+```
+需要注意的是，其中 `tidb_get_token_duration` 的单位是微秒（us）， 其 `COMMENT` 里面也有提到。
```
Comment (lonng) [inline session3/chapter3/metrics-summary.md]:
```suggestion
> 注意：其中 `tidb_get_token_duration` 的单位是微秒（us）， 其 `COMMENT` 里面也有提到。
```

File: session3/chapter3/metrics-summary.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+```
+
+上面查询结果表示：
+
+- t2 时间段内的 tidb_slow_query_cop_process_total_time （tidb 慢查询中的 cop process 耗时 ） 比 t1 时间段高了 5865 倍
+- t2 时间段内的 tidb_distsql_partial_scan_key_total_num（tidb 的 distsql 请求扫描key 的数量） 比 t1 时间段高了 3648 倍
+- t2 时间段内的 tikv_cop_total_response_size（tikv 的 cop 请求结果的大小 ） 比 t1 时间段高了 192 倍
+- t2 时间段内的 tikv_cop_scan_details（tikv 的 cop 请求的 scan ） 比 t1 时间段高了 105 倍
+
+通过上面2个对比查询可以大致了解集群在这2个时间段的负载情况。t2 时间段的 Cop 请求要比 t2 时间段高很多，TiKV 的 Copprocessor 有些处理不过来，出现了一些 cop task 等待，可以猜测可能是 t2 时间段出现了一些大查询，或者是查询较多的负载。
```
Comment (lonng) [inline session3/chapter3/metrics-summary.md]:
```suggestion
通过上面两个时间段对比查询可以大致了解集群在这2个时间段的负载情况。t2 时间段的 Cop 请求要比 t2 时间段高很多，导致 TiKV 的 Copprocessor 过载，出现了 cop task 等待，可以猜测可能是 t2 时间段出现了一些大查询，或者是查询较多的负载。
```

**Review by lonng — COMMENTED** [review]
Rest LGTM

**Review by lonng — APPROVED** [review]
LGTM

**Review by AilinKid — COMMENTED** [review]
LGTM

**Pull request merged**